scalanet/src/io/iohk/scalanet/peergroup/kademlia/KMessage.scala

```diff
@@ -0,0 +1,28 @@
+package io.iohk.scalanet.peergroup.kademlia
+
+import java.util.UUID
+
+import scodec.bits.BitVector
+
+sealed trait KMessage[V] {
+  def requestId: UUID
+  def nodeId: BitVector
+  def nodeRecord: V
+}
+
+object KMessage {
```

Comment:
do we plan to implement PING, FIND_NODE and STORE?

Reply:
We plan PING and FIND_NODE.